core/tests/protractor_utils/general.js

```diff
@@ -66,6 +66,8 @@ var SERVER_URL_PREFIX = 'http://localhost:9001';
 var LIBRARY_URL_SUFFIX = '/library';
 var EDITOR_URL_SLICE = '/create/';
 var PLAYER_URL_SLICE = '/explore/';
+var CREATOR_DASHBOARD = '/dashboard';
```

Comment:
Variable name should be CREATOR_DASHBOARD_URL or similar -- the string fragment is really a URL, it isn't actually the dashboard. Ditto below.